src/xarray_regrid/methods/conservative.py

```diff
@@ -66,7 +66,7 @@ def conservative_regrid(
     # Attempt to infer the latitude coordinate
     if latitude_coord is None:
         for coord in data.coords:
-            if str(coord).lower() in ["lat", "latitude"]:
+            if str(coord).lower().startswith("lat"):
```

Comment:
I had purposefully made it check only if the coordinate is `lat` or `latitude`. If it's something different, the user can specify it.

Reply:
Sure, happy to revert this. I have occasionally seen for example `long` as a coordinate, but this could theoretically catch unwanted similar names as well.